app/static/js/base.js

```diff
@@ -15,8 +18,11 @@ $(document).ready(function() {
     $("#userSelect").on('change', function() {
         $("#userSelectForm").submit();
     });
+    $("#flashes").delay(5000).fadeOut();
```

Comment:
This is the right idea, but it will only work for one, because it assumes each id is unique